.travis.yml

```diff
@@ -18,7 +18,10 @@ matrix:
     - env: TOXENV=format
     - env: TOXENV=mypy
     - env: TOXENV=docs
-cache: pip
+cache:
+  - pip: true
+    directories:
+      - "$HOME/build"
```

Comment:
Is this caching now also the compiled libs?

Reply:
Yes. I don't think this is harmful. There's a separate cache for each platform and language version. (Or at least that's what the travis-ci documentation leads me to believe.)